src/main/java/org/folio/list/services/CustomTenantService.java

```diff
@@ -1,37 +1,138 @@
 package org.folio.list.services;
 
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import lombok.extern.log4j.Log4j2;
+import org.folio.list.domain.ListEntity;
+import org.folio.list.repository.ListRepository;
 import org.folio.spring.FolioExecutionContext;
 import org.folio.spring.liquibase.FolioSpringLiquibase;
 import org.folio.spring.service.PrepareSystemUserService;
 import org.folio.spring.service.TenantService;
 import org.folio.tenant.domain.dto.TenantAttributes;
+import org.jetbrains.annotations.NotNull;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Primary;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Service;
 
+import java.util.*;
+
 @Log4j2
 @Primary
 @Service
 public class CustomTenantService extends TenantService {
 
   protected final PrepareSystemUserService prepareSystemUserService;
+  private final ListRepository listRepository;
+  @Autowired
+  private ObjectMapper objectMapper;
 
   @Autowired
   public CustomTenantService(
     JdbcTemplate jdbcTemplate,
     FolioExecutionContext context,
     FolioSpringLiquibase folioSpringLiquibase,
-    PrepareSystemUserService prepareSystemUserService
+    PrepareSystemUserService prepareSystemUserService,
+    ListRepository listRepository
   ) {
     super(jdbcTemplate, context, folioSpringLiquibase);
     this.prepareSystemUserService = prepareSystemUserService;
+    this.listRepository = listRepository;
   }
 
   @Override
   protected void afterTenantUpdate(TenantAttributes tenantAttributes) {
     log.info("Initializing system user");
-    prepareSystemUserService.setupSystemUser();
+   prepareSystemUserService.setupSystemUser();
+    Collection<ListEntity> lists = (Collection<ListEntity>) listRepository.findAll();
```

Comment:
Do we really need this cast? `findAll()` returns an `Iterable`, so it should be usable directly in the enhanced for-loop without needing to cast it